src/main/java/bio/terra/cli/serialization/persisted/PDWorkspace.java

```diff
@@ -88,6 +93,11 @@ public Builder googleProjectId(String googleProjectId) {
       return this;
     }
 
+    public Builder property(Properties property) {
```

Comment:
this should be `properties`. Note that this is our last chance to change it since it's a persisted field.